runtime/Stdlib_Array.resi

```diff
@@ -302,6 +302,9 @@ external splice: (array<'a>, ~start: int, ~remove: int, ~insert: array<'a>) => u
 external toSpliced: (array<'a>, ~start: int, ~remove: int, ~insert: array<'a>) => array<'a> =
   "toSpliced"
 
+@send
+external removeInPlace: (array<'a>, int, @as(1) _) => unit = "splice"
+
```

Comment:
nitpicking, but wouldn't it be better to move the tests here as docstring tests? At least they would help documenting the function.

Whatever we decide to do, I'd add some documentation here anyway, at least to document that the second argument is an index. Maybe it's a sign we should use a labelled argument here (`~start`, `~index`?) like the other functions with indices here, what do you guys think? @zth ?

Reply:
Added a documentation comment 👍

Regarding labeled arguments I think it's consistent for the array API to not use it when we have a single argument which is an index. But it would definitely make sense if we had more arguments like:

```
@send
external removeMultipleInPlace: (array<'a>, ~start: int, ~number: int) => unit = "splice"
```

Reply:
many functions with just one int parameter in the Stdlib.Array module are labeled, it might be worth making all of this more consistant, but this could definitely be in another PR.